CSS/ShapeStyle.css

```diff
@@ -463,6 +463,9 @@ Footer {
     .leftAddition, .rightAddition{;
         z-index: -10;
     }
+        .leftAddition.containsLink, .rightAddition.containsLink {
```

Comment:
mixed tabs/spaces indentation?

Reply:
Matter of code preferences, though changed in c55bb91 so that it is in line with the rest of the file.